pymc3/sampling.py

```diff
@@ -480,7 +480,12 @@ def _update_start_vals(a, b, model):
             if is_transformed_name(tname) and get_untransformed_name(tname) == name:
                 transform_func = [d.transformation for d in model.deterministics if d.name == name]
                 if transform_func:
-                    b[tname] = transform_func[0].forward(a[name]).eval()
```

Comment:
Try it out please. I was confused by the shortcut `pymc3.sampling` above the codeblock and thought it is internals of sampling. Seems like it can be the right solution as it is executed once.